notes/stories/film/film-scifi-1990s.st.txt

```diff
@@ -10571,6 +10571,46 @@ father and son [we saw brief flashbacks to Truman with his dad who went missing
 mother and son [we saw Truman with his mom from time to time]
 
 
+movie: The XFiles Fight the Future (1998)
+=========================================
+
+:: Title
+The X-Files: Fight the Future
+
+:: Description
+The X-Files (also known as The X-Files: Fight the Future) is a 1998 American
+science fiction thriller film based on Chris Carter's television series of
+the same name, which revolves around fictional unsolved cases called the
+X-Files and the characters solving them.
+
+:: Date
+1998-06-19
+
+:: References
+https://en.wikipedia.org/wiki/The_X-Files_(film)
+
+:: Ratings
+3 <paul>
+
+:: Genre
+sci-fi
+
+:: Major Themes
+secret society conspiracy theory [The story follows the FBI agents Mulder and Skully as they attempt to get to the bottom of a global conspiracy being perpetrated by an unnamed shadowy organization to prepare the way for Earth to be colonized by space aliens.]
+subversive life form from outer space [A shadowy organization with a global presence was preparing the way for space aliens to colonize Earth.]
+extraterrestrial being [Powerful space aliens were using a shadowy organization to spread a virus around the world as a prelude to colonization.]
+human characters in contrast [The intuition-trusting, paranormal-believing Agent Mulder was pointedly juxtaposed with his rational-minded, skeptical partner Agent Skully.]
```

Comment:
Typo

Reply:
My brain isn't seeing it. Where is it?

Reply:
My bad, I thought it was meant to be institution-trusting but I see now it reads intuition-trusting.

Reply:
Ah, it's funny you mention so, because I made the same association myself at one point.